chef/data_bags/crowbar/bc-template-nova.json

```diff
@@ -31,8 +31,7 @@
         "database": "nova"
       },
       "rbd": {
-        "user": "",
-        "secret_uuid": ""
+        "user": ""
```

Comment:
No, this value should stay empty if we would like to use multiple rbd backends in cinder